gapis/resolve/filter.go

```diff
@@ -70,5 +70,26 @@ func buildFilter(
 			return (cmd.CmdFlags().IsExecutedDraw() && len(idx) > 1) || cmd.CmdFlags().IsSubmission()
 		})
 	}
+	if f.GetSuppressHostCommands() {
+		filters = append(filters, func(id api.CmdID, cmd api.Cmd, s *api.GlobalState, idx api.SubCmdIdx) bool {
+			return cmd.CmdFlags().IsSubmission()
+		})
+	}
+	if f.GetSuppressBeginEndMarkers() {
+		filters = append(filters, func(id api.CmdID, cmd api.Cmd, s *api.GlobalState, idx api.SubCmdIdx) bool {
+			return !(cmd.CmdFlags().IsPushUserMarker() || cmd.CmdFlags().IsPopUserMarker() || cmd.CmdFlags().IsUserMarker())
```

Comment:
This should also filter out the `vkBeginRenderPass` and `vkEndRenderPass` commands, as well as the ones that start/end the transform feedback grouping. Essentially, all the commands that are implied by a group starting/ending.

Reply:
Again, I suspect this means adding more custom labels to commands? Is there any way of working this using the existing labels?

Reply:
I don't think there is anything currently.

Reply:
Added a new label for this @begin_end_renderpass